server/src/main/java/io/crate/metadata/functions/Signature.java

```diff
@@ -138,15 +138,15 @@ private static Signature.Builder signatureBuilder(FunctionName name, FunctionTyp
         return builder;
     }
 
-    public static Builder builder() {
+    static Builder builder() {
         return new Builder();
     }
 
-    public static Builder builder(Signature signature) {
+    private static Builder builder(Signature signature) {
         return new Builder(signature);
     }
 
-    public static class Builder {
+    protected static class Builder {
```

Comment:
Will eventually make it private just would like a quick review before fixing remaining tests.